datadog/resource_datadog_security_monitoring_rule.go

```diff
@@ -108,6 +116,30 @@ func datadogSecurityMonitoringRuleSchema() map[string]*schema.Schema {
 						Required:         true,
 						Description:      "A signal will “close” regardless of the query being matched once the time exceeds the maximum duration. This time is calculated from the first seen timestamp.",
 					},
+
+					"new_value_options": {
```

Comment:
`new_value` would make more sense imo since this is already inside of `options`

Reply:
hmm; agreed; but it's the same "string" in the json api; so like this we have a direct mapping; I think it will help avoiding confusions.